.eslintrc.js

```diff
@@ -21,4 +21,17 @@ module.exports = {
 			],
 		},
 	},
+	rules: {
+		'jsdoc/check-tag-names': [
+			'warn',
+			{
+				definedTags: [
+					'values', // required for vue-styleguideist
+					'notExported',
+					'packageDocumentation',
+					'jest-environment',
```

Comment:
Jest is not used anymore. Can we get rid of this then?

Reply:
Yes, fixed :)